rosdep/base.yaml

```diff
@@ -800,6 +800,8 @@ ffmpeg2theora:
   fedora: [ffmpeg2theora]
   gentoo: [media-video/ffmpeg2theora]
   ubuntu: [ffmpeg2theora]
+flac:
```

Comment:
please add fedra/gentoo ... and also you'd better to add more information to help maintainer's decision https://github.com/ros/rosdistro/pull/19652

Reply:
Thank you for your advice!
I updated the comment.